domain.md

```diff
@@ -0,0 +1,71 @@
+In order to deploy CAM that is integrated with customer own Active Directory (AD), the folloing requirements must be met. When deploying CAM, the **account credential and DC fqdn** must be provided, and the DC must be able to communicated with the given **fqdn**.
+
+# 1) Creating an account with delegated permissions to join a computer to domain. 
+
+The following steps setup an account with delegated privileges to join a computer to domain, assuming an account already created, otherwise, please create an account first.
+
+ * Login Domain Controller Server.
+ * Lanuch Active Directory Users and Computers.
+ * Locate the **OU** that you want to join a computer to and right-click and then click Delegate Control. 
+ * The **Delegation of Control Wizard** opens, click **Next**
+ * In the **Users or Groups** page, click **Add** to add a specific user or a specific group to the **Selected users and groups** list, and then click **Next**
+ * In the **Tasks to Delegate** page, click **Create a custom task to delegate**, and then click **Next**.
+ * In the **Active Directory Object Type** page, Click **Only the following objects in the folder**, and then from the list, click to select the **Computer objects** check box. Then, select the check boxes below the list, **Create selected objects in this folder** and **Delete selected objects in this folder**, Click **Next**.
+ * In the **Permissions** page, click to select the following permission check boxes from list:  
```

Comment:
Which one allows unlimited domain joins?

Reply:
Once delegated permission assign to the account, the limitation is removed

Reply:
Great!